### PR TITLE
Extracted unlocked-did into an environment var

### PR DESCRIPTION
**Pull request by rhysyngsun**
Part of #1 

#### What changed

This extracts the statically loaded `unloaded-did*.json` file into a base64-encoded environment variable.

Also a few other things that were done to support this:

- Extracted `process.env` parsing into `config.ts`
- Added tests for the new code
- Updated existing `require()` usages to use `import` instead (a few `@types/*` dependencies added to support this)
- Tweaked the docker-compose setup to support iterative testing without having to rebuild the container

#### How to verify

Run the example requests in the README, they should still work as expected.

### CODE REVIEW

**Comment by aliceriot** [comment]
is the plan to continue to support running the service locally, for development, outside of docker-compose? If so I think a few tweaks will be needed, possibly adding something like [dotenv](https://www.npmjs.com/package/dotenv) to parse a `.env` file for configuration.

**Review by aliceriot — CHANGES_REQUESTED** [review]
works great with the docker setup, just had a few questions / comments on things

File: types/vc-js/index.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+declare module 'vc-js';
```
[the remark this answers is not in the document]
Comment (rhysyngsun) [reply types/vc-js/index.d.ts:1]: Yeah, typescript fails on them when you `import` them. Previously they were `require`'d, which typescript doesn't type-check. These defs just basically type the whole module as `any`.

[the remark this answers is not in the document]
Comment (aliceriot) [reply types/vc-js/index.d.ts:1]: gotcha, makes sense!

**Review by kimdhamilton — APPROVED** [review]

File: src/config.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+let CONFIG: null | Config = null;
+
+export function parseConfig(): Config {
+  return Object.freeze({
+    port: process.env.PORT ? Number(process.env.PORT) : 5000,
+    unlockedDid: process.env.UNLOCKED_DID ? JSON.parse(
+      Buffer.from(process.env.UNLOCKED_DID, "base64").toString("ascii")
+    ): null
+
```
Comment (aliceriot) [inline src/config.ts:16]: stray empty line? we might wanna throw prettier on here at some point

**Review by aliceriot — APPROVED** [review]
just noticed one little formatting thing, otherwise looks good!

**Pull request merged**